src/long.ts

```diff
@@ -291,6 +330,41 @@ export class Long extends BSONValue {
     return result;
   }
 
+  /**
+   * Returns a Long representation of the given string, written using the specified radix.
+   * Throws an error if any of the following conditions are true:
+   * - the string contains invalid characters for the given radix
+   * - the string contains whitespace
+   * - the value the string represents is too large or too small to be a Long
+   * @param str - The textual representation of the Long
+   * @param unsigned - Whether unsigned or not, defaults to signed
+   * @param radix - The radix in which the text is written (2-36), defaults to 10
+   * @returns The corresponding Long value
+   */
+  static fromStringStrict(str: string, unsigned?: boolean, radix?: number): Long {
```

Comment:
```suggestion
  static fromStringStrict(string: string, unsigned?: boolean, radix?: number): Long {
```

This will be in our public API, let's make sure we use descriptive variable names since that's what shows up in autocomplete and in our docs.

(optional) I also tend to avoid abbreviations inside our code for the same reason it's best practice to ensure we use good, full parameter names in our public API.  So I'd use `cleanedString` instead of `cleanedStr`.  But this is mostly preference, feel free to ignore this part.

Reply:
oh, interesting.  I didn't notice that.  we can leave it as-is then